handler.ts

```diff
@@ -126,7 +126,10 @@ export const makeGeocoderRequests = async (
     )
   }
 
-  const query = new URLSearchParams(event.queryStringParameters).toString()
+  // Pelias has different layers, and so needs to ignore the layers parameter
+  // if it is present
+  const peliasQSP = { ...event.queryStringParameters }
```

Comment:
Maybe call this "custom QSP" since it's possible for the primary geocoder to be pelias too.

Reply:
One-liner equivalent:

```suggestion
  const { layers, ...peliasQSP } = event.queryStringParameters
```

Reply:
Doesn't this make it less clear what's happening? Since we wouldn't be using the `layers`